pandas/core/dtypes/api.py

```diff
@@ -59,6 +60,7 @@
     "is_float",
     "is_float_dtype",
     "is_hashable",
+    "is_implicit_conversion_to_float64",
```

Comment:
To fix the original issue, we shouldn't need to create a new function (and expose it as a public API)